specs/marketplace.md

```diff
@@ -152,27 +138,30 @@ The the table below provides the description of the `Request` and the associated
 | `client` | `address` | The Codex node requesting storage. |
 | `ask` | `Ask` | Parameters of Request. |
 | `content` | `Content` | The dataset that will be hosted with the storage request. |
-| `expiry` | `uint256` | Timeout in seconds during which all the slots have to be filled, otherwise Request will get cancelled. The final deadline timestamp is calculated at the moment the transaction is mined. |
+| `expiry` | `uint256` | Timeout in seconds during which all the slots have to be filled, otherwise request will get cancelled. The final deadline timestamp is calculated at the moment the transaction is mined. |
 | `nonce` | `byte32` | Random value to differentiate from other requests of same parameters. It SHOULD be a random byte array. |
-| `reward` | `uint256` | Amount of tokens that will be awarded to SPs for finishing the storage request. It MUST be an amount of Tokens offered per slot per second. The Ethereum address that submits the `requestStorage()` transaction MUST have [approval](https://docs.openzeppelin.com/contracts/2.x/api/token/erc20#IERC20-approve-address-uint256-) for the transfer of at least an equivalent amount in Tokens. |
```

Comment:
This is obvious, I do not think it should be capitalized.

Reply:
Okey, sounds good.

Reply:
I agree this should be capitalised because "Tokens" is a proper noun in this case. We are using it in lieu of "CDX" or "Codex tokens" since this is not legally approved, and may change in the future.

Reply:
So you are saying keep it uppercased? Then lets revert the lower-casing of the Definition table as well?

Reply:
Hmm, I really don't know what the orthodoxy is with a definitions table in a spec. My comment refers to the capitalization of the word "Tokens" in a sentence. In which case, it is *generally* (but not always) a proper noun.

Reply:
I asked Jimmy to lowercase all the "definitions" in the Definition table because of lowercasing of the "Token": https://github.com/codex-storage/codex-spec/pull/7/files#r1785680631

Reply:
@emizzle This decision is really a matter of preference. I suggested lowercasing because the definition defines an ERC-20 token used with Codex and is the only token type mentioned in the spec. This makes the mention of "token" to be a generic ERC-20 token.  This spec does not introduce or define a "CDX/Codex token".  In this context, tokens should be capitalized and the Codex token should be defined (contract address, symbol).